cli/src/org/partiql/shell/ShellHighlighter.kt

```diff
@@ -20,17 +20,17 @@ import org.jline.reader.LineReader
 import org.jline.utils.AttributedString
 import org.jline.utils.AttributedStyle
 import java.io.PrintStream
-import java.nio.file.Path
+import java.net.URI
 import java.util.regex.Pattern
 
 private val SUCCESS: AttributedStyle = AttributedStyle.DEFAULT.foreground(AttributedStyle.GREEN)
 private val ERROR: AttributedStyle = AttributedStyle.DEFAULT.foreground(AttributedStyle.RED)
 private val INFO: AttributedStyle = AttributedStyle.DEFAULT
 private val WARN: AttributedStyle = AttributedStyle.DEFAULT.foreground(AttributedStyle.YELLOW)
 
-class ShellHighlighter(private val syntaxFile: Path) : Highlighter {
+class ShellHighlighter(syntaxUri: URI) : Highlighter {
```

Comment:
Somewhat related to this PR's changes but does the `ShellHighlighter` need to be public?